packages/config/config/devices/0x018c/pm-b400zw-n.json

```
@@ -0,0 +1,73 @@
+{
+	"manufacturer": "Dawon DNS",
+	"manufacturerId": "0x018c",
+	"label": "PM-B400ZW-N",
+	"description": "Power Manager",
```

```suggestion
	"description": "Smart Plug with Power Meter",
```
sounds more like what the device does.

---

ok